ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -88,6 +88,11 @@
   public abstract void onBackPressed();
 
   /**
+   * This method will give JS the opportunity to receive intents via Linking.
+   */
+  public abstract void onNewIntent(Intent intent);
```

Comment:
I think this is good here. We should implement this for the new bridge as well (you can consider current bridge is deprecated). That would mean copying the code from `ReactInstanceManagerImpl` to https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/XReactInstanceManagerImpl.java.